File: dataBinding/bindingLayoutAndBindingStaticData/app/src/main/java/com/example/bindinglayoutandbindingstaticdata/viewmodels/MainViewModel.java
package com.example.bindinglayoutandbindingstaticdata.viewmodels;

import android.app.Application;

import androidx.annotation.NonNull;
import androidx.lifecycle.AndroidViewModel;

import com.example.bindinglayoutandbindingstaticdata.model.User;

public class MainViewModel extends AndroidViewModel {

    private User mUser;

    public MainViewModel(@NonNull Application application) {
        super(application);

        mUser = new User("Ali", 35);
    }

    public User getUser() {
        return this.mUser;
    }
}

File: dataBinding/importInDataBinding/app/src/main/java/com/example/databindingfragment/viewmodels/MainViewModel.kt
package com.example.databindingfragment.viewmodels

import android.app.Application
import androidx.lifecycle.AndroidViewModel
import com.example.databindingfragment.model.DataProvider
import com.example.databindingfragment.model.Product

class MainViewModel(application: Application) : AndroidViewModel(application) {

    private var product = DataProvider.productList.get(5)

    fun getProduct(): Product = product

}
